matter: merge upstream commits up to 9012f0 (#5707)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Matter (Project CHIP)
=====================

The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``9012f08de9b7340e7d59d51a7ec8a6cdcfda9d15``.

The following list summarizes the most important changes inherited from the upstream Matter:

* Added:

>   * Support for Administrator Commissioning Cluster which allows to enable or disable the commissioning window on a Matter device.
>     It is required by the Matter multi-admin functionality.

Reviewer: ```suggestion
  * Support for Administrator Commissioning Cluster, which allows enabling or disabling the commissioning window on a Matter device.
    This is required by the Matter multi-admin functionality.
```
